tests/unit/config_test.py

```diff
@@ -239,6 +239,21 @@ def test_config_extra_hosts_list_of_dicts_validation_error(self):
                 )
             )
 
+    def test_config_allows_either_type(self):
```

Comment:
It was supposed to be generic, I'm trying not to fall into trap of testing every single schema field, more covering each aspect of the different kinds of schema functionality. So this could be expose or another field where the definition states that it can be either of 2 things. So really I'm testing `"type": ["something", "somethingelse"]` does what I expect it to do in the schema.